inbc-program/README.md

```diff
@@ -65,11 +65,7 @@ Ensure trusted repository in intel_manageability.conf is to be configured with t
 ### Usage
 ```
 inbc fota {--uri, -u=URI}  
-   [--releasedate, -r RELEASE_DATE; default="2026-12-31"] 
-   [--vendor, -v VENDOR; default="Intel Corporation"] 
-   [--biosversion, -b BIOS_VERSION; default="ADLSFWI1.R00"] 
-   [--manufacturer, -m MANUFACTURER; default="Intel Corporation"] 
-   [--product, -pr PRODUCT; default="Alder Lake Client Platform"] 
+   [--releasedate, -r RELEASE_DATE; default="2026-12-31"]   
```

Comment:
Just to confirm. We still use the releasedate but not rest of the parameters (e.g. vendor, manufacturer etc) is not needed?
Is that what is seen from the logs in the bug opened by validation?